### PR TITLE
Correct Vite configuration to fix broken GitHub Pages demo

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: online/vite.config.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import { defineConfig } from "vite";
-import react from "@vitejs/plugin-react";
+import rescript from "@jihchi/vite-plugin-rescript";
 
-// https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()],
-  base: "/rescript-json-schema",
+  base: process.env.CI ? "https://dzakh.github.io/rescript-json-schema/" : "/",
```
Comment (DZakh) [inline online/vite.config.js]: Is the change needed? If yes, maybe it should be something like this?
```suggestion
  base: process.env.CI ? "/rescript-json-schema/" : "/",
```

File: online/package.json [hunk 2]
```diff
@@ -3,31 +3,24 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "res:build": "rescript build",
-    "res:clean": "rescript clean",
-    "res": "rescript build -w",
     "vendor": "rsync -av --exclude='node_modules' --exclude='lib' --exclude='coverage' --exclude='__tests__' --exclude='.git' --exclude='online' ../ ./vendor/rescript-json-schema",
     "dev": "vite",
```
Comment (DZakh) [inline online/package.json:7]: Is the compiler built-in in the Vite plugin?

**Review by DZakh — COMMENTED** [review]
Thank you for your contribution. Looks good, but I have a few questions about the solution.

File: online/package.json (hunk 2 above)
Comment (smorimoto) [reply online/package.json:7]: Yes

File: online/vite.config.js (hunk 1 above)
Comment (smorimoto) [reply online/vite.config.js]: It might still work, but I don’t have any confidence 🤷‍♂️

Comment (smorimoto) [reply online/vite.config.js]: I just confirmed that it works in my repository, so I updated the PR.

**Comment by DZakh** [comment]
Oops, I forgot about the PR. Thank you for your contribution 🔥

**Pull request merged**